bindings/python/gpt4all/gpt4all.py

```diff
@@ -0,0 +1,70 @@
+"""
+Python only API for running all GPT4All models.
+"""
+import os
+
+from . import pyllmodel
+
+
+class GPT4All():
```

Comment:
I want one high-level method here called chat_completion which has the exact same data format as open ai.
```python
# Note: you need to be using OpenAI Python v0.27.0 for the code below to work
import openai

openai.ChatCompletion.create(
  model="gpt-3.5-turbo",
  messages=[
        {"role": "system", "content": "You are a helpful assistant."},
        {"role": "user", "content": "Who won the world series in 2020?"},
        {"role": "assistant", "content": "The Los Angeles Dodgers won the World Series in 2020."},
        {"role": "user", "content": "Where was it played?"}
    ]
)

```
It should accepted a list of dictionaries with the  keys,values in `messages` and internally call generate with the proper formatting